gamebench_api_client/models/dataframes/generic/abstract_generic.py

```diff
@@ -12,6 +12,8 @@ class AbstractGenericModel(AbstractModel, ABC):
         individually as needed.
     """
 
+    METRIC_PATH = ''
```

Comment:
@tristan-fickes-bfg 
Could this be moved to abstract_model.AbstractModel? And do we need the session_detail package at all anymore?

Reply:
Nope I don't think anything is blocking it from being set in the AbstractModel class.  I will do that in a bit!  As for the session_detail package, that is fully removed in the #30 branch as that was the branch where we created the attributes on SessionSummary so the user can access that information.

Reply:
The AbstractModel class now creates the METRIC_PATH variable.